libs/platform/experience/src/controllers/layout.controller.ts

```diff
@@ -5,7 +5,15 @@ import {
 } from '@spryker-oryx/experience/layout';
 import { Size, featureVersion, sizes } from '@spryker-oryx/utilities';
 import { LitElement, TemplateResult, html } from 'lit';
-import { Observable, map, withLatestFrom } from 'rxjs';
+import {
```

Comment:
There's a featureVersion usage in this file that conflicts with https://spryker.slack.com/archives/C029XPSBXC6/p1697630001287519. Can you make it consistent?